pkg/secretless/config/v2/README.md

```diff
@@ -0,0 +1,77 @@
+##Overview
```

Comment:
I think the `##` pattern usually assumes a space after (e.g. `## Overview` instead of `##Overview`. The rest of the headers might need that change too.

Reply:
Readme is a bit less readable in content than maybe we should have it (it makes sense to secretless devs but it won't be to newcomers) but it's not a blocker